### PR TITLE
Build/manager tools

### PR DESCRIPTION
**Pull request by jleveugle**
Add the following packages:

* component-rollup-config
* webpack-config
* webpack-dev-server

All of theses 3 packages has been migrated to TypeScript, to reinforce codebase and deliver better dist.

Installation of theses packages forces to clean some other dependencies, like `less` dependencies.

In fact, `less@^2.0.0` is completely removed, and we will use `less@^3.9.0` instead.

### CODE REVIEW

File: packages/manager/tools/component-rollup-config/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  plugins: ['@typescript-eslint'],
```
Comment (marie-j) [inline packages/manager/tools/component-rollup-config/.eslintrc.js]: 😮

File: packages/manager/tools/component-rollup-config/src/plugins/translation-utils.ts [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  'es_US',
+  'fi_FI',
+  'fr_CA',
+  'fr_FR',
+  'it_IT',
+  'lt_LT',
+  'nl_NL',
+  'pl_PL',
+  'pt_PT',
+];
```
Comment (marie-j) [inline packages/manager/tools/component-rollup-config/src/plugins/translation-utils.ts]: Does it make sense to have a `module` made of constants ? I think there's also a list in `manager-core`

File: packages/manager/tools/webpack-dev-server/src/sso/index.ts [hunk 3]
```diff
@@ -0,0 +1,148 @@
+/* eslint-disable no-console */
+import assign from 'lodash/assign';
+import base64url from 'base64url';
+import cookie from 'cookie';
+import proxy from 'request';
+
+interface Config {
```
Comment (marie-j) [inline packages/manager/tools/webpack-dev-server/src/sso/index.ts]: 🎉

File: packages/manager/tools/component-rollup-config/LICENSE [hunk 4]
```diff
@@ -13,14 +13,14 @@ modification, are permitted provided that the following conditions are met:
   this list of conditions and the following disclaimer in the documentation
```
Comment (antleblanc) [inline packages/manager/tools/component-rollup-config/LICENSE]: This file mustn't be moved from `packages/manager/apps/pci/`.

Please prefer add a new `LICENSE` file to the new module.

File: packages/manager/tools/component-rollup-config/README.md [hunk 5]
```diff
@@ -0,0 +1,206 @@
+# Component rollup config
+
+> Extensible rollup configuration to build OVH components.
+
+[![Downloads](https://badgen.net/npm/dt/@ovh-ux/component-rollup-config)](https://npmjs.com/package/@ovh-ux/component-rollup-config) [![Dependencies](https://badgen.net/david/dep/ovh-ux/component-rollup-config)](https://npmjs.com/package/@ovh-ux/component-rollup-config?activeTab=dependencies) [![Dev Dependencies](https://badgen.net/david/dev/ovh-ux/component-rollup-config)](https://npmjs.com/package/@ovh-ux/component-rollup-config?activeTab=dependencies) [![Gitter](https://badgen.net/badge/gitter/ovh-ux/blue?icon=gitter)](https://gitter.im/ovh/ux)
```
Comment (antleblanc) [inline packages/manager/tools/component-rollup-config/README.md]: Both `Dependencies` and `Dev Dependencies` must be updated (due to the new path inside the monorepo).

**Example for `Dependencies`:**
`https://badgen.net/david/dep/ovh-ux/component-rollup-config` > `https://badgen.net/david/dep/ovh-ux/manager/packages/manager/tools/component-rollup-config`

File: packages/manager/tools/component-rollup-config/README.md [hunk 6]
```diff
@@ -0,0 +1,206 @@
...
+yarn test
+```
+
+## Related
+
+* [rollup-plugin-less-tilde-importer](https://github.com/ovh-ux/rollup-plugin-less-tilde-importer) - Rollup plugin to facilitate less imports with a ~ (tilde) prefix
+
+## Contributing
+
+Always feel free to help out! Whether it's [filing bugs and feature requests](https://github.com/ovh-ux/component-rollup-config/issues/new) or working on some of the [open issues](https://github.com/ovh-ux/component-rollup-config/issues), our [contributing guide](CONTRIBUTING.md) will help get you started.
```
Comment (antleblanc) [inline packages/manager/tools/component-rollup-config/README.md]:
```suggestion
Always feel free to help out! Whether it's [filing bugs and feature requests](https://github.com/ovh-ux/manager/issues/new) or working on some of the [open issues](https://github.com/ovh-ux/manager/issues), our [contributing guide](CONTRIBUTING.md) will help get you started.
```

File: packages/manager/tools/component-rollup-config/tsconfig.json [hunk 7]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (antleblanc) [inline packages/manager/tools/component-rollup-config/tsconfig.json]: Couldn't be inherit from a root configuration?

I mean creating a new package that export this configuration for instance.

File: packages/manager/tools/webpack-config/README.md [hunk 8]
```diff
@@ -0,0 +1,80 @@
+# Manager Webpack Config
+
+> Extensible webpack configuration for the OVH Manager.
+
+[![Downloads](https://badgen.net/npm/dt/@ovh-ux/manager-webpack-config)](https://npmjs.com/package/@ovh-ux/manager-webpack-config) [![Dependencies](https://badgen.net/david/dep/ovh-ux/manager-webpack-config)](https://npmjs.com/package/@ovh-ux/manager-webpack-config?activeTab=dependencies) [![Dev Dependencies](https://badgen.net/david/dev/ovh-ux/manager-webpack-config)](https://npmjs.com/package/@ovh-ux/manager-webpack-config?activeTab=dependencies) [![Gitter](https://badgen.net/badge/gitter/ovh-ux/blue?icon=gitter)](https://gitter.im/ovh/ux)
```
Comment (antleblanc) [inline packages/manager/tools/webpack-config/README.md]: Please refer to previous comment about badges path (component-rollup-config).

File: packages/manager/tools/webpack-config/README.md [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+});
+```
+
+## Related
+
+* [manager-webpack-dev-server](https://github.com/ovh-ux/manager-webpack-dev-server) - OVH manager webpack development server configuration
+
+## Contributing
+
+Always feel free to help out! Whether it's [filing bugs and feature requests](https://github.com/ovh-ux/manager-webpack-config/issues/new) or working on some of the [open issues](https://github.com/ovh-ux/manager-webpack-config/issues), our [contributing guide](CONTRIBUTING.md) will help get you started.
```
Comment (antleblanc) [inline packages/manager/tools/webpack-config/README.md]:
```suggestion
Always feel free to help out! Whether it's [filing bugs and feature requests](https://github.com/ovh-ux/manager/issues/new) or working on some of the [open issues](https://github.com/ovh-ux/manager/issues), our [contributing guide](CONTRIBUTING.md) will help get you started.
```

File: packages/manager/tools/webpack-dev-server/README.md [hunk 10]
```diff
@@ -0,0 +1,56 @@
+# Manager Webpack Dev Server
+
+> OVH manager webpack development server configuration.
+
+[![Downloads](https://badgen.net/npm/dt/@ovh-ux/manager-webpack-dev-server)](https://npmjs.com/package/@ovh-ux/manager-webpack-dev-server) [![Dependencies](https://badgen.net/david/dep/ovh-ux/manager-webpack-dev-server)](https://npmjs.com/package/@ovh-ux/manager-webpack-dev-server?activeTab=dependencies) [![Dev Dependencies](https://badgen.net/david/dev/ovh-ux/manager-webpack-dev-server)](https://npmjs.com/package/@ovh-ux/manager-webpack-dev-server?activeTab=dependencies) [![Gitter](https://badgen.net/badge/gitter/ovh-ux/blue?icon=gitter)](https://gitter.im/ovh/ux)
```
Comment (antleblanc) [inline packages/manager/tools/webpack-dev-server/README.md]: Please refer to previous comment about badges path (component-rollup-config).

File: packages/manager/tools/webpack-dev-server/README.md [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+};
+```
+
+## Related
+
+* [manager-webpack-config](https://github.com/ovh-ux/manager-webpack-config) - OVH manager shared webpack configuration
+
+## Contributing
+
+Always feel free to help out! Whether it's [filing bugs and feature requests](https://github.com/ovh-ux/manager-webpack-dev-server/issues/new) or working on some of the [open issues](https://github.com/ovh-ux/manager-webpack-dev-server/issues), our [contributing guide](CONTRIBUTING.md) will help get you started.
```
Comment (antleblanc) [inline packages/manager/tools/webpack-dev-server/README.md]:
```suggestion
Always feel free to help out! Whether it's [filing bugs and feature requests](https://github.com/ovh-ux/manager/issues/new) or working on some of the [open issues](https://github.com/ovh-ux/manager/issues), our [contributing guide](CONTRIBUTING.md) will help get you started.
```

File: packages/manager/tools/webpack-config/package.json [hunk 12]
```diff
@@ -0,0 +1,79 @@
...
+  "devDependencies": {
+    "@babel/core": "^7.2.2",
+    "@babel/generator": "^7.2.2",
+    "@babel/parser": "^7.2.3",
+    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
+    "@babel/preset-env": "^7.2.3",
+    "@commitlint/cli": "^7.5.2",
+    "@commitlint/config-angular": "^7.5.0",
+    "@ovh-ux/component-rollup-config": "^5.1.0",
+    "@ovh-ux/manager-webpack-dev-server": "^2.1.0",
```
Comment (antleblanc) [inline packages/manager/tools/webpack-config/package.json]: Pay attention. An issue has been opened to provide the latest version of `@ovh-ux/manager-webpack-dev-server`.

Here you are importing the latest version that hasn't be tested with the actual configuration (https://github.com/ovh-ux/manager/pull/986/files?file-filters%5B%5D=.json&file-filters%5B%5D=.lock&file-filters%5B%5D=.md#diff-4722f9be2756722c66d3df3f9a4f947bR3)

ref: https://github.com/ovh-ux/manager-webpack-config/issues/74

File: packages/manager/tools/component-rollup-config/.eslintrc.js (hunk 1 above)
Comment (antleblanc) [reply packages/manager/tools/component-rollup-config/.eslintrc.js]: @marie-j Could you please be more precise about your reaction?

Thanks!

File: packages/manager/tools/component-rollup-config/README.md [hunk 13]
```diff
@@ -0,0 +1,206 @@
...
+yarn test
+```
+
+## Related
+
+* [rollup-plugin-less-tilde-importer](https://github.com/ovh-ux/rollup-plugin-less-tilde-importer) - Rollup plugin to facilitate less imports with a ~ (tilde) prefix
+
+## Contributing
+
+Always feel free to help out! Whether it's [filing bugs and feature requests](https://github.com/ovh-ux/manager/issues/new) or working on some of the [open issues](https://github.com/ovh-ux/component-rollup-config/issues), our [contributing guide](CONTRIBUTING.md) will help get you started.
```
Comment (antleblanc) [inline packages/manager/tools/component-rollup-config/README.md]:
```suggestion
Always feel free to help out! Whether it's [filing bugs and feature requests](https://github.com/ovh-ux/manager/issues/new) or working on some of the [open issues](https://github.com/ovh-ux/manager/issues), our [contributing guide](CONTRIBUTING.md) will help get you started.
```

File: packages/manager/tools/webpack-config/README.md [hunk 14]
```diff
@@ -0,0 +1,80 @@
...
+});
+```
+
+## Related
+
+* [manager-webpack-dev-server](https://github.com/ovh-ux/manager-webpack-dev-server) - OVH manager webpack development server configuration
+
+## Contributing
+
+Always feel free to help out! Whether it's [filing bugs and feature requests](https://github.com/ovh-ux/manager/issues/new) or working on some of the [open issues](https://github.com/ovh-ux/manager-webpack-config/issues), our [contributing guide](CONTRIBUTING.md) will help get you started.
```
Comment (antleblanc) [inline packages/manager/tools/webpack-config/README.md]:
```suggestion
Always feel free to help out! Whether it's [filing bugs and feature requests](https://github.com/ovh-ux/manager/issues/new) or working on some of the [open issues](https://github.com/ovh-ux/manager/issues), our [contributing guide](CONTRIBUTING.md) will help get you started.
```

File: packages/manager/tools/webpack-dev-server/README.md [hunk 15]
```diff
@@ -0,0 +1,56 @@
...
+};
+```
+
+## Related
+
+* [manager-webpack-config](https://github.com/ovh-ux/manager-webpack-config) - OVH manager shared webpack configuration
+
+## Contributing
+
+Always feel free to help out! Whether it's [filing bugs and feature requests](https://github.com/ovh-ux/manager/issues/new) or working on some of the [open issues](https://github.com/ovh-ux/manager-webpack-dev-server/issues), our [contributing guide](CONTRIBUTING.md) will help get you started.
```
Comment (antleblanc) [inline packages/manager/tools/webpack-dev-server/README.md]:
```suggestion
Always feel free to help out! Whether it's [filing bugs and feature requests](https://github.com/ovh-ux/manager/issues/new) or working on some of the [open issues](https://github.com/ovh-ux/manager/issues), our [contributing guide](CONTRIBUTING.md) will help get you started.
```

File: packages/manager/tools/component-rollup-config/.eslintrc.js (hunk 1 above)
Comment (antleblanc) [reply packages/manager/tools/component-rollup-config/.eslintrc.js]: Why conversation has been resolved?

Meaning any preferences related to: https://github.com/palantir/tslint?

**Pull request merged**